### ISSUE TITLE
Dynamic opacity not working.

### ISSUE THREAD

**Issue by pradhanharshil** [micheleg/dash-to-dock#946]
Dynamic opacity is not working in the latest version of dash to dock. When I select option dynamic it automatically gets switched to fixed.

Distro: Arch Linux
Gnome 3.32

**Comment by willmtemple** [comment]
Workaround:

```
$ gsettings --schemadir ~/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/schemas set org.gnome.shell.extensions.dash-to-dock transparency-mode "DYNAMIC"
```

Once I set transparency-mode manually, I get dynamic opacity back and I can use the gear icon in the extension settings to configure it, though the dropdown combo is unpopulated.

**Comment by pradhanharshil** [comment]
It worked. thanks for the solution!

**Comment by memeplex** [comment]
This is a workaround, not a proper solution, please reopen this issue since it's a real problem.

**Comment by pradhanharshil** [comment]
@memeplex Sorry for the late reply.
yes, you are right. This is a real problem which needs to be reviewed by the developer. Hence, I am reopening this issue.

Thanks.

**Comment by dvorapa** [comment]
Is this still an issue in the last version?

**Comment by willmtemple** [comment]
@dvorapa yes

**Comment by dvorapa** [comment]
Yeah, for Gnome 3.32 there will most likely be no update, the last version 67 is only for Gnome 3.34, but you can try to update from master
